gateway-service-admin/src/main/java/org/apache/knox/gateway/service/admin/TopologiesResource.java

```diff
@@ -360,7 +361,8 @@ public Response uploadProviderConfiguration(@PathParam("name") String name, @Con
         }
       } catch (URISyntaxException e) {
         log.invalidResourceURI(e.getInput(), e.getReason(), e);
-        response = status(Response.Status.BAD_REQUEST).entity("{ \"error\" : \"Failed to deploy provider configuration " + name + "\" }").build();
+        response = status(Response.Status.BAD_REQUEST).entity("{ \"error\" : \"Failed to deploy provider configuration " + Encode
+            .forHtml(name) + "\" }").build();
```

Comment:
Seems like a weird break? Put `Encode.forHtml(...` on the same line?

Reply:
yeah, I fixed one weird break and introduced another :(